CharlesProxy/CharlesProxy.munki.recipe

```diff
@@ -26,6 +26,13 @@
             <string>%NAME%</string>
             <key>unattended_install</key>
             <true/>
+            <key>postinstall_script</key>
+            <string>#!/bin/sh
+
+echo "Setting permissions on Charles.app..."
+chmod -R g+w "$3/Applications/Charles.app/Contents/MacOS/Charles Proxy Settings"
```

Comment:
Remember the $3 issue for the future, though!

-Greg

> On Oct 12, 2015, at 1:32 PM, Elliot Jordan <notifications@github.com> wrote:
> 
> In CharlesProxy/CharlesProxy.munki.recipe <https://github.com/autopkg/arubdesu-recipes/pull/30#discussion_r41798053>:
> 
> > @@ -26,6 +26,13 @@
> >              <string>%NAME%</string>
> >              <key>unattended_install</key>
> >              <true/>
> > +            <key>postinstall_script</key>
> > +            <string>#!/bin/sh
> > +
> > +echo "Setting permissions on Charles.app..."
> > +chmod -R g+w "$3/Applications/Charles.app/Contents/MacOS/Charles Proxy Settings"
> Noted, but moot now since I removed the postinstall_script.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/autopkg/arubdesu-recipes/pull/30/files#r41798053>.
>